Pyrado/pyrado/algorithms/step_based/sac.py

```diff
@@ -349,8 +350,8 @@ def update(self):
         self.logger.add_value("avg expl strat std", to.mean(expl_strat_stds))
         self.logger.add_value("ent_coeff", self.ent_coeff)
         if self._lr_scheduler_policy is not None:
-            self.logger.add_value("avg lr policy", to.mean(self._lr_scheduler_policy.get_last_lr()), 6)
-            self.logger.add_value("avg lr critic", to.mean(self._lr_scheduler_qfcns.get_last_lr()), 6)
+            self.logger.add_value("avg lr policy", np.mean(self._lr_scheduler_policy.get_last_lr()), 6)
+            self.logger.add_value("avg lr critic", np.mean(self._lr_scheduler_qfcns.get_last_lr()), 6)
```

Comment:
Would also the buildin mean() function work?

Reply:
If you mean `statistics.mean` then probably yes, although I do not see why this should be used instead of `np.mean`.

Reply:
ignore that :)